python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -35,11 +36,10 @@ class LocalFile(IGangaFile):
                                      'subfiles': ComponentItem(category='gangafiles', defvalue=[], hidden=1,
                                                 sequence=1, copyable=0, doc="collected files from the wildcard namePattern"),
                                      'compressed': SimpleItem(defvalue=False, typelist=[bool], protected=0, doc='wheather the output file should be compressed before sending somewhere'),
-                                     #'output_location': SimpleItem(defvalue=None, typelist=[str, None], hidden=1, copyable=1, doc="path of output location on disk")
                                      })
     _category = 'gangafiles'
     _name = "LocalFile"
-    _exportmethods = ["location", "remove", "accessURL"]
+    _exportmethods = ["location", "remove", "accessURL", "copyTo"]
```

Comment:
It would be nice in general to have derived Ganga classes add to the `_exportmethods` list rather than just overwrite it but that's for another time. I agree that it should be exposed in the other files explicitly for the moment though.